locations/storefinders/yext_answers.py

```diff
@@ -0,0 +1,74 @@
+import json
+from typing import Any, Iterable
+from urllib.parse import urlencode
+
+from scrapy import Request
+from scrapy.http import JsonRequest, Response
+from scrapy.spiders import Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+
+class YextAnswersSpider(Spider):
+    dataset_attributes = {"source": "api", "api": "yext"}
+
+    endpoint: str = "https://liveapi.yext.com/v2/accounts/me/answers/vertical/query"
+    api_key: str = ""
+    api_version: str = "20220511"
+    page_limit: int = 50
+
+    def make_request(self, offset: int) -> JsonRequest:
+        return JsonRequest(
+            url="{}?{}".format(
+                self.endpoint,
+                urlencode(
+                    {
+                        "experienceKey": "tgi-fridays-search",
```

Comment:
`experienceKey` should be another API parameter?

Reply:
Done, thanks.